package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "group-income",
   "version": "0.0.1",
-  "contractsVersion": "0.0.7",
+  "contractsVersion": "0.0.10",
```

Comment:
We jumped from `.7` to `.10`?

I think this should be `.9`, right? (now that `.8` has been merged)

Reply:
[This PR](https://github.com/okTurtles/group-income/pull/1434) will make the contract version of 0.0.9